pkg/assetfetcher/config.go

```diff
@@ -0,0 +1,35 @@
+package assetfetcher
+
+import "context"
+
+// AssetFetcherConfig holds configuration to connect to Nexpose
+// and make a call to the fetch assets API
+type AssetFetcherConfig struct {
+	Host                 string
```

Comment:
Suggestion: You can add help information to these with a tag like:

```golang
type AssetFetcherConfig struct {
  Host string `description:"The URL of a Nexpose API endpoint."`
}
```

Reply:
That's neat, I'll add descriptions